mypy/checkexpr.py

```diff
@@ -1233,7 +1233,7 @@ def check_for_extra_actual_arguments(self,
         for i, kind in enumerate(actual_kinds):
             if i not in all_actuals and (
                     kind != nodes.ARG_STAR or
-                    not is_empty_tuple(actual_types[i])):
+                    is_not_empty_tuple(actual_types[i])):
```

Comment:
Here it would be nice to add a comment about list and `Any` star args potentially being empty, so we don't complain about them.

Reply:
I added the comment.